sdk/template/template/package.json

```diff
@@ -33,7 +33,7 @@
     "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-test/index.node.js",
+    "unit-test:node": "env TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' mocha --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace test/node/index.spec.ts",
```

Comment:
having to set the compiler option here feels pretty bad

Reply:
Yeah, this isn't actually necessary and should be removed. It's taken care of with `mocha -r esm`, and the other packages show that instead of this. Will make this change.

I _really_ wanted to avoid another tsconfig, thankfully we don't have to do that either.